gateway-service-impl/src/main/java/org/hypertrace/gateway/service/common/converters/LongToAttributeKindConverter.java

```diff
@@ -26,6 +26,11 @@ public Value doConvert(Long value, AttributeKind attributeKind, Value.Builder va
         valueBuilder.setString(value.toString());
         return valueBuilder.build();
 
+      case TYPE_TIMESTAMP:
```

Comment:
Tip: This is a PR that mostly includes version updates and unrelated changes, I would suggest in this kind of PRs we point out what is the important change. I guess this is the important change in this PR.